src/tlog-test-fd-json-reader.c

```diff
@@ -121,9 +121,10 @@ test(const char *n, const struct test t)
     }
 
 #define FAIL(_fmt, _args...) \
-    do {                                                \
-        fprintf(stderr, "%s: " _fmt "\n", n, ##_args);  \
-        passed = false;                                 \
+    do {                                              \
+        fprintf(stderr, "FAIL %s:%d: %s " _fmt "\n", \
```

Comment:
Please align the backslash at the end of this line horizontally with all the others in the macro.